Docker/solver/dockerfile

```diff
@@ -1,24 +1,24 @@
 # Use the pre-configured image from Docker Hub as the base
 FROM simvascular/libraries:ubuntu22 AS builder
 
-# Get latest svFSIplus solver from GitHub repository
-RUN git clone https://github.com/SimVascular/svFSIplus.git
+# Get latest svMultiPhysics solver from GitHub repository
+RUN git clone https://github.com/SimVascular/svMultiPhysics.git
 
 # Trilinos build
-WORKDIR /svFSIplus/build-trilinos
+WORKDIR /svMultiPhysicsbuild-trilinos
```

Comment:
there is a missing / (svMultiPhysics/build-trilinos)

Reply:
@dcodoni Good eye, fixed!